package.json

```diff
@@ -63,6 +63,9 @@
   },
   "dependencies": {
     "@renderlesskit/react": "^0.2.0",
+    "@types/classnames": "^2.2.11",
+    "classnames": "^2.2.6",
+    "date-fns": "^2.16.1",
```

Comment:
Remove all three !!

May be move `date-fns` to dev-deps & move the customInput example to Storybook 🤔